CONTRIBUTING.md

```diff
@@ -59,6 +61,10 @@ Updating an existing devfile stack is relatively straightforward:
     - Minimally, testing with odo v2 (`odo create`, `odo push`, etc) and odo v3 (`odo init`, `odo dev`, etc) is recommended, however if your Devfile is used with other tools, it's recommended to test there as well.
 5) Open a pull request against this repository with your changes.
 
+### Automatic Stack Image Update
+
+As images used inside the stacks need to be up-to-date and in order to avoid using the `latest` tag, the renovate bot runs periodically ensuring that all images used from stacks (for example, inside components) have the latest version. As a result, all images used inside a devfile of a stack need to have a fixed version (e.g `1.1.0`) instead of `latest`.
```

Comment:
```suggestion
Avoid using the `latest` tag, the renovate bot runs periodically ensuring that all image references used from stacks (for example, inside components) have the latest version. As a result, all image references used inside the stack devfiles will need to have a fixed version (e.g `1.1.0`) instead of `latest`.
```